src/main/java/com/magicvet/service/ClientService.java

```diff
@@ -29,6 +30,20 @@ public Client registerNewClient(){
         return client;
     }
 
+    public void addPet(Client client, PetService petService) {
+        System.out.print("Do you want to add a new pet now? (y/n): ");
+        String answerAddPet = Main.SCANNER.nextLine();
+        if (answerAddPet.toLowerCase().trim().startsWith("y")) {
+            System.out.println("Adding a new pet.");
+            Pet pet = petService.registerNewPet();
+            client.setPet(pet);
+            pet.setOwnerName(client.getFirstName() + " " + client.getLastName());
+            System.out.println("Pet has been added.");
+        } else {
+            System.out.println("Have a nice day!");
+        }
+    }
```

Comment:
What happens if the client inputs wrong data except y or n? The current implementation of the addPet method does not handle it. Let's enhance the method to handle invalid input by repeatedly prompting the user until valid input is provided. You can achieve this by using a loop.

Reply:
According to the reviewer's suggestion, the check of the client's input was added to the addPet method.

Reply:
Everything looks good! You can merge it into the main branch.